lib/starknet_explorer_web/live/transaction_live.ex

```diff
@@ -582,11 +586,19 @@ defmodule StarknetExplorerWeb.TransactionLive do
       |> Enum.reject(&is_nil/1)
 
     # change fee formatting
-    actual_fee = BlockUtils.format_hex_for_display(transaction.receipt.actual_fee)
-    max_fee = BlockUtils.format_hex_for_display(transaction.max_fee)
+    actual_fee = Utils.hex_wei_to_eth(transaction.receipt.actual_fee)
+
+    transaction =
+      case transaction.type != "L1_HANDLER" do
```

Comment:
Is't more idiomatic?
```
case transaction.type do
   "L1_HANDLER" -> ...
   _ -> ...
```

Reply:
Ah yes, this was an `if` which I decided to change but only did it half-way